packages/atomic/src/components/search/atomic-result-multi-value-text/atomic-result-multi-value-text.spec.ts

```diff
@@ -0,0 +1,329 @@
+    expect(more).toBeNull();
+  });
+
+  it('should truncate values when array exceeds max', async () => {
+    const {allValues, more} = await renderComponent({
+      resultState: {
+        raw: {tags: ['tag1', 'tag2', 'tag3', 'tag4', 'tag5'], urihash: ''},
+      },
+      maxValuesToDisplay: 3,
+    });
+
+    expect(allValues).toHaveLength(3);
+    expect(more).toHaveTextContent('2 more');
+  });
+
+  it('should render separators between values', async () => {
+    const {allSeparators} = await renderComponent({
+      resultState: {
+        raw: {tags: ['tag1', 'tag2', 'tag3', 'tag4', 'tag5'], urihash: ''},
+      },
+      maxValuesToDisplay: 3,
+    });
+
+    expect(allSeparators).toHaveLength(3);
+  });
+
+  it('should show all values in more label when maxValuesToDisplay is 0', async () => {
+    const {allValues, more} = await renderComponent({
+      resultState: {
+        raw: {tags: ['tag1', 'tag2', 'tag3', 'tag4', 'tag5'], urihash: ''},
+      },
+      maxValuesToDisplay: 0,
+    });
+
+    expect(allValues).toHaveLength(0);
+    if (more) {
+      expect(more).toHaveTextContent('5 more');
+    }
+  });
+
+  it('should split string by delimiter', async () => {
+    const {allValues} = await renderComponent({
+      resultState: {
+        raw: {categories: 'cat1;cat2;cat3', urihash: ''},
+      },
+      field: 'categories',
+      delimiter: ';',
+    });
+
+    expect(allValues).toHaveLength(3);
+    expect(allValues[0]).toHaveTextContent('cat1');
+    expect(allValues[1]).toHaveTextContent('cat2');
+    expect(allValues[2]).toHaveTextContent('cat3');
+  });
+
+  it('should treat as single value when no delimiter specified', async () => {
+    const {allValues} = await renderComponent({
+      resultState: {
+        raw: {categories: 'cat1;cat2;cat3', urihash: ''},
+      },
+      field: 'categories',
+    });
+
+    expect(allValues).toHaveLength(1);
+    expect(allValues[0]).toHaveTextContent('cat1;cat2;cat3');
+  });
+
+  it('should render single value', async () => {
+    const {allValues, more} = await renderComponent({
+      resultState: {
+        raw: {single_value: 'single', urihash: ''},
+      },
+      field: 'single_value',
+    });
+
+    expect(allValues).toHaveLength(1);
+    expect(more).toBeNull();
+    expect(allValues[0]).toHaveTextContent('single');
+  });
+
+  it('should render nothing when field value is empty string', async () => {
+    const {element} = await renderComponent({
+      resultState: {
+        raw: {empty_field: '', urihash: ''},
+      },
+      field: 'empty_field',
+    });
+
+    expect(element).toBeEmptyDOMElement();
+  });
+
+  it('should render nothing when field does not exist', async () => {
+    const {element} = await renderComponent({
+      resultState: {
+        raw: {urihash: ''},
+      },
+      field: 'nonexistent_field',
+    });
+
+    expect(element).toBeEmptyDOMElement();
+  });
+
+  describe('when field value is not a string or array', () => {
```

Comment:
No need for a describe wrapper / beforeEach here; there's just one `it` so it should be

"should set error when field value is not a string or array"